src/framework/preferences/js/PrefsEditor.js

```diff
@@ -385,7 +390,7 @@ var fluid_2_0 = fluid_2_0 || {};
      * Resets the selections to the integrator's defaults and fires onReset
      */
     fluid.prefs.prefsEditor.reset = function (that) {
-        that.updateModel(fluid.copy(that.rootModel));
+        that.applier.change("", fluid.copy(that.rootModel));
```

Comment:
Leave a note here that this may not be completely effective if the root model is smaller than the current one. Given our previous discoveries re "model shrinkage" (http://issues.fluidproject.org/browse/FLUID-5585 ), the proper thing to do here is to apply a DELETE to the root before putting in the new model. And this should be done within a transaction in order to avoid notifying the tree more than necessary. However, the transactional model of the changeApplier is going to change radically soon (re. "New New Notes") and this implementation doesn't seem to be causing a problem at present so we had just better leave it the way it is for now, with a comment pointing to the potential problem and requirements.